pages/index.tsx

```diff
@@ -92,7 +96,7 @@ const LastBlockInfo = () => {
   )
 }
 
-const LatestBlocks = () => {
+const LatestBlocks = ({ t }: { t: Translator }) => {
```

Comment:
Do you know if the `useTranslation` hook does any sort of caching or batch loading of the translation files? It seems like it would be nice to do `useTranslation` in each component directly rather than passing `t` through props, since the components will rely on keys from specific translation files.

Reply:
💯 agree, I went down the props-passing-path last night when I couldn't figure out how to debug things. Now I think I've encapsulated each component separately.